jsonifier/models/models.py

```diff
@@ -105,6 +105,13 @@ def _jsonify_record(self, parser, rec, root):
                     value, json_key = self._jsonify_record_handle_resolver(
                         rec, field, resolver, json_key
                     )
+            # whatever json value we have found in subparser or not ass a sister key
+            # on the same level {jsoni_key}_fieldname
+            if rec.env.context.get("with_fieldname"):
+                json_key_fieldname = "_".join(["fieldname", json_key])
```

Comment:
Would it make sense to prefix the whole key with `_` as well? In the odd case that some Odoo implementation has both `foo` and `fieldname_foo` as fields in a model